AdyenActions/Components/QRCode/PixComponent.swift

```diff
@@ -17,8 +17,8 @@ internal enum QRCodeComponentError: LocalizedError {
     }
 }
 
-/// A component that presents a QR code.
-public final class QRCodeComponent: ActionComponent, Cancellable {
+/// A component  for Pix payment method.
+public final class PixComponent: ActionComponent, Cancellable {
```

Comment:
This could be confusing: this is not a "A component  for Pix payment method.", this is "A component  for Pix action."

I suggest we rename it to `PixActionComponent` and then hide it behind `QRCodeComponent`